dss/index/bundle.py

```diff
@@ -71,10 +79,14 @@ def _read_file_infos(cls, replica: Replica, fqid: BundleFQID, manifest: JSON) ->
                                    f"not be parsed. This file will not be indexed. Exception: {ex}")
                     continue
                 except BlobStoreError as ex:
-                    exception_type = type(ex).__name__
-                    logger.warning(f"In bundle {fqid} the file '{file_name}' is marked for indexing yet could "
-                                   f"not be accessed. This file will not be indexed. Exception: {exception_type}, "
-                                   f"file blob key: {file_blob_key}")
+                    if context.get_remaining_time_in_millis() > time_needed:
+                        logger.warning(f"In bundle {fqid} the file '{file_name}' is marked for indexing yet could "
+                                       f"not be accessed. Retrying.")
+                        files_info.append(file_info)
+                        time.sleep(time_wait)
```

Comment:
time.sleep(time_wait)
check if there is still time, continue
Make raise the default behavior.